manifests/prepare/package.pp

```diff
@@ -17,20 +17,25 @@
   if $puppet_agent::params::master_agent_version != $::aio_agent_version {
     $pe_server_version = pe_build_version()
 
-    $source = "puppet:///pe_packages/${pe_server_version}/${::platform_tag}/${package_file_name}"
+    if $::osfamily == 'windows' {
+      $tag = $::puppet_agent::arch ? {
+        'x64' => 'windows-x86_64',
+      }
+      $source = "puppet:///pe_packages/${pe_server_version}/${tag}/${package_file_name}"
```

Comment:
So that is a new behavior that was introduced in 2015.3.2, before `pe_repo` would just copy the agent into the ${tag} directory, now we add the version. On platforms that use `manifests/repo.pp` we use https://github.com/puppetlabs/puppetlabs-pe_repo/blob/2015.3.x/manifests/repo.pp#L104 to setup a symlink to that tag from the old path. Windows doesn't have that symlink.